run_linux.go

```diff
@@ -1806,6 +1806,20 @@ func (b *Builder) runSetupVolumeMounts(mountLabel string, volumeMounts []string,
 			case "private", "rprivate", "slave", "rslave", "shared", "rshared":
 				rootProp = opt
 			}
+
+			if strings.Contains(opt, "upperdir") {
```

Comment:
Shouldn’t this be `HasPrefix`? A check for `HasPrefix(…, "upperdir=")` could even avoid the need for the other check.

(One could over-engineer this and parse every entry into a key[=value], and then do the switch purely on key values. Or maybe the switch should be `case opt == "rw":… case strings.HasPrefix(opt, "upperdir")`. Up to the maintainers of the repo.)

Reply:
Swtiched to hasprefix.